sdk/src/Core/AWSSDK.Core.NetStandard.csproj

```diff
@@ -23,6 +23,8 @@
     <NoWarn>$(NoWarn);CS1591;CA1822</NoWarn>
 
     <SignAssembly>True</SignAssembly>
+
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Like discussed before this is required for some of the shim APIs.

Reply:
I don't understand this argument
The net runtime allows unsafe and runs pretty much on any platform. Azure SDK allows unsafe and runs on a variety of platforms too. Rabbitmq client which I contribute too is used in all sorts of environments and allows unsafe too. With the choice of your TFMs you will have to backport some amount of helpers that sometimes require "unsafe" modifier in order to balance getting perf and also reducing the complexity of the code base by reducing the number of redundant code paths that have to be implemented across the board.

Unsafe is unsafe in the sense that you need to know what you are doing and yes ultimately there should be as little as possible unsafe code but that should not mean there can't be any because of a fear of making a mistake.

Fyi the code here was ported from NET originally and brought into the Azure SDK and has been battle tested against a plethora of targets over years